app/src/services/elastic.js

```diff
@@ -668,6 +668,7 @@ export default class ElasticClient {
     async searchHPODescendants( hpo ) {
         const uri = `${this.host}/${esIndices.hpo}/_search`
         const body = {
+            size: 1000,
```

Comment:
why hardcoded here instead of a default value as param?

Reply:
Ill add it in a constant. Its not something we will like to pass in a parameter in this endpoint. Its just to make sure that all kids a returned for a phenotype (1000 is more than enough in all cases).